modules/javafx.controls/src/test/java/test/javafx/scene/control/ListViewTest.java

```diff
@@ -2665,4 +2698,9 @@ public void fixListViewCrash_JDK_8303680() {
         // We don't check for the position of the cell, because it's currently don't work properly.
         // But we wan't to ensure, that the VirtualFlow "Doesn't crash" - which was the case before.
     }
+
+    private static double toViewPortLength(double prefHeight) {
+        // it would be better to calculate this from listView but there is no API for this
+        return prefHeight - 2;
```

Comment:
I think after this node is inside the scene tree and has a skin, you can just calculate that with: 
`prefHeight(-1) - snappedTopInset() - snappedBottomInset()`. 
Something like that I already used.

Also, `Viewport` is one word, so should not be camel case.

Reply:
using `prefHeight(-1) - snappedTopInset() - snappedBottomInset()` breaks the original test, I'd rather avoid that (all I did was minor refactoring to move the logic and the comment into one place).

thanks for -> `toViewportLength()`